test cases/linuxlike/11 runpath rpath ldlibrarypath/meson.build

```diff
@@ -0,0 +1,14 @@
+project('runpath rpath ldlibrarypath', 'c')
+
+libsrc = files('lib.c')
+
+subdir('lib1')
+subdir('lib2')
+
+lib2dir = meson.current_build_dir() + '/lib2'
```

Comment:
This is not correct on all backends or when using a flat layout. The correct thing is to do `current_build_dir` in the subdir. However this test probably requires specific subdir layout so just add a comment about it above this line.